resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/rest/kubernetes/v2/ResourceStagingServerSslOptionsProviderSuite.scala

```diff
@@ -87,6 +87,32 @@ class ResourceStagingServerSslOptionsProviderSuite extends SparkFunSuite with Be
     }
   }
 
+  test("Setting pem files without setting passwords should use random passwords.") {
```

Comment:
The existing tests all use keyStores and trustStores and we would need to write a new test to cover using PEM files instead.